lib/shared/drive-constraints.js

```diff
@@ -208,3 +208,162 @@ exports.isDriveValid = (drive, image) => {
 exports.isDriveSizeRecommended = (drive, image) => {
   return _.get(drive, [ 'size' ], UNKNOWN_SIZE) >= _.get(image, [ 'recommendedDriveSize' ], UNKNOWN_SIZE);
 };
+
+/**
+ * @summary Drive/image compatibility status messages.
+ * @public
+ * @type {Object}
+ *
+ * @description
+ * Status messages intended to be shown to the user.
+ */
+exports.COMPATIBILITY_STATUS_MESSAGES = {
+
+  /**
+   * @property {String} OK
+   * @memberof COMPATIBILITY_STATUS_MESSAGES
+   *
+   * @description
+   * The image and drive compatibility is OK.
+   */
+  OK: '',
```

Comment:
It's slightly "symmetry-breaking", but rather than assigning an empty string to a not-actually-used-anywhere constant, I guess we could define the status objects returned by `getDriveImageCompatibilityStatus` to only have a `.message` attribute when the `.type` is `WARNING` or `ERROR` ?